curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/quoting-strings-with-single-quotes.english.md

```diff
@@ -43,7 +43,7 @@ Right now, the <code>&#60;a&#62;</code> tag in the string uses double quotes eve
 
 ```yml
 tests:
-  - text: Remove all the <code>backslashes</code> (<code>\</code>)
+  - text: You should remove all the <code>backslashes</code> (<code>\</code>).
     testString: assert(!/\\/g.test(code) && myStr.match('\\s*<a href\\s*=\\s*"http://www.example.com"\\s*target\\s*=\\s*"_blank">\\s*Link\\s*</a>\\s*'));
   - text: You should have two single quotes <code>&#39;</code> and four double quotes <code>&quot;</code>
```

Comment:
```suggestion
  - text: You should have two single quotes <code>&#39;</code> and four double quotes <code>&quot;</code>.
```

Reply:
@moT01 Can not commit suggested change "as-is" because you have deleted a space off the beginning of the line which must be there.